res/values-fr/strings.xml

```diff
@@ -58,8 +58,8 @@
     <string name="SERVER">"SERVEUR"</string>
     <string name="Joining_game_in_">"Rejoindre la partie "</string>
     <string name="Game_Full">"Partie complète"</string>
-    <string name="US_East">"Amérique Centrale"</string>
-    <string name="US_West">"Amérique du Nord"</string>
+    <string name="US_East">"Etats-Unis de l'Est"</string>
+    <string name="US_West">"Etats-Unis de l'Ouest"</string>
```

Comment:
The server was really changed its name, so the previous name was correct.

Reply:
What do you mean by currently changed ? Because you can see that the translation is not correct, in english i just saw that servers are US east and west but it's still confused with America central and north which do not match at all 
Can you tell me a little more in detail ? ^^
And thanks for reply

Reply:
**Shot 1:** English in-game
**Shot 2:** English strings file

![Screenshot_20240819_180310_Nebulousio.jpg](https://github.![Screenshot_20240819_180357_GitHub.jpg](https://github.com/user-attachments/assets/14f602a3-b513-4379-ba18-c4ec712070d8)

49ba449c)

![Uploading Screenshot_20240819_180357_GitHub.jpg …]()

Reply:
**Shot 1:** English in-game
**Shot 2:** English strings file

![Screenshot_20240819_180310_Nebulousio.jpg](https://github.com/user-attachments/assets/fd375636-b273-463d-926f-d685d0e89e0b)

![Screenshot_20240819_180357_GitHub.jpg](https://github.com/user-attachments/assets/aeba12d0-d3da-4e8e-ab15-ee36c86de82e)

Reply:
It changed on the update ? Because i see name in english ![image](https://github.com/user-attachments/assets/d50c263d-dc38-46a5-a41f-3f4b3469f19e)

And i just notice that it was Etat-Unis in french version ![image](https://github.com/user-attachments/assets/cbdca0de-ee21-4292-8f80-c6374f109db5)

Reply:
Yes, you need to update the game to be able to see the changes.

Reply:
It's not available on iOS but im gonna change it back 
Thanks

Reply:
It was on the previous version 8.0.5 ( IOS ) on beta version but now it is fixed with the new update